python/docs/source/user_guide/sql/index.rst

```diff
@@ -16,13 +16,14 @@
     under the License.
 
 
-=========
-Spark SQL
-=========
+=================
+DataFrame and SQL
```

Comment:
I think we should be consistent with the main PySpark doc page terminology now that we are changing this:
https://spark.apache.org/docs/latest/api/python/index.html

On the page, we say "Spark SQL and DataFrames"
I suggest we do the same here.